en-US/win10/SetupTPM.md

```diff
@@ -136,22 +136,22 @@ Firmware TPM (fTPM) requires special Processor/SoC support and whence fTPM is no
 ##<a name="dTPM"></a>Setup dTPM  
 These instructions are applicable for any dTPM module supported on either MBM or RPi2.
 
-1. Get a discrete TPM module and attach it to the MBM.
-2. Based on your discrete TPM module of choice, identify its matching ACPI table [here][3].
-3. Enable testsigning on the device:
-
-        bcdedit /set {current} integrityservices disable
-        bcdedit /set testsigning on
-
-4. Copy the ACPI table to MBM _C:\Windows\System32\ACPITABL.dat_.
-5. Disable fTPM by changing the following UEFI settings:
+1. Get a discrete TPM module and attach it to the MBM/RPi2.
+2. (Applies to MBM) Disable fTPM by changing the following UEFI settings:
 
         Device Manager -> System Setup -> Security Configuration -> PTT = <Disable>
 
-6. Enable dTPM by changing the following UEFI settings:
+3. (Applies to MBM) Enable dTPM by changing the following UEFI settings:
 
         Device Manager -> System Setup -> Security Configuration -> Discrete TPM = <Enable>
 
+4. Based on your discrete TPM module of choice, identify its matching ACPI table [here][3].
+5. Copy that ACPI table to MBM/RPi2 _C:\Windows\System32\ACPITABL.dat_.
+6. Enable testsigning on the device:
+
+        bcdedit /set {current} integrityservices disable
+        bcdedit /set testsigning on
+
 7. Reboot the device.
 8. Verify you have the right TPM version enabled - run the [TPM 2.0 Tool][1] on the Windows IoT Core device.
```

Comment:
The footnote link model isn't recommended. you should link it normally using markdown syntax.